### DISCUSSION TITLE
[Dica] Material sobre UX Design em pt-BR

### DISCUSSION THREAD

**Discussion by devigor** [frontendbr/forum#546] [Dicas]
Fala pessoal!!!
Tempos atrás eu estava começando a estudar UX Design e resolvi pedi ajuda aqui no fórum, como muitos me ajudaram eu resolvi vir aqui e dar minha contribuição para vocês.
Eu monteu uma lista com diversos links sobre o assunto, links em pt-BR que eu espero que ajudem quem está começando na área.

Caso tem alguma contribuição a fazer, seu PR será bem vindo!!
Não esquece de deixar aquela estrelinha para ajudar o projeto!

 Link: https://github.com/devigor/awesome-ux-ui

**Comment by phfedev** [comment]
Valeu @devigor
